docs/product-user-guide/src/main/asciidoc/assets-types-ref.adoc

```diff
@@ -1,23 +1,91 @@
+* DRL file
++
+A rule file is typically a file with a .drl extension.
+In a DRL file you can have multiple rules, queries and functions, as well as some resource declarations like imports, globals and attributes that are assigned and used by your rules and queries.
+However, you are also able to spread your rules across multiple rule files (in that case, the extension .rule is suggested, but not required) - spreading rules across files can help with managing large numbers of rules. A DRL file is simply a text file.
+
+* DSL definition
++
+Domain Specific Languages (DSLs) are rule languages that are dedicated to problem domains.
+
+* Enumeration
++
+Data enumerations are an optional asset type that can be configured to provide drop down lists for the guided designer. They are stored and edited just like any other asset, and apply to the package that they belong to.
+
+* Form
++
+Forms are used for collecting user data for business process. {CENTRAL} provides the option to automatically generate forms, which can then be edited to meet specific business process requirements.
+
+* Global Variable(s)
++
+Global variables are used to make application objects available to the rules. Typically, they are used to provide data or services that the rules use, especially application services used in rule consequences, and to return data from the rules, like logs or values added in rule consequences, or for the rules to interact with the application, doing callbacks.
+
+* Guided Decision Table
++
+Decision tables are collections of rules stored in either a spreadsheet or in the {PRODUCT_DM} user interface as guided decision tables.
+
+* Guided Decision Table Graph
++
+A Guided Decision Table Graph is a collection of related guided decision tables that are displayed within a single designer. You can use this designer to better visualize and work with various related decision tables in one location. Additionally, when a condition or an action in one table uses the same data type as a condition or an action in another table, the tables will be physically linked with a line in the table graph designer.
++
+For example, if one decision table determines a loan application rate and another table uses the application rate to determine some other action, then the two decision tables are linked in a guided decision table graph.
+
+* Guided Rule
++
+Rules provide the logic for the rule engine to execute against.
+A rule includes a name, attributes, a `when` statement on the left hand side of the rule, and a `then` statement on the right hand side of the rule.
+
+* Guided Rule Template
++
+Guided rule templates provide a reusable rule structure for multiple rules that are compiled into Drools Rule Language (DRL) and form the core of the decision service for your project.
+
+* Package
++
+All assets are contained in packages in Drools Workbench. A package is like a folder (it also serves as a "namespace"). A home folder for rule assets to live in.
```

Comment:
IMHO, Drools Workbench is community naming

Reply:
Changed to:

All assets are contained in packages in {CENTRAL}. A package is a folder for rules and also serves as a "namespace".